src/plugins/vuetify.ts

```diff
@@ -11,8 +11,13 @@ import 'vuetify/styles'
 // Composables
 import { createVuetify } from 'vuetify'
 
+import { VDataTableVirtual } from 'vuetify/labs/VDataTable'
```

Comment:
Data tables are seemingly not yet fully baked into Vuetify 3 so we have to import them from 'labs'. See https://github.com/vuetifyjs/vuetify/issues/13479

Reply:
Data tables! Mad science!